test/functional/plugin/lsp_spec.lua

```diff
@@ -624,6 +624,37 @@ describe('LSP', function()
       eq(true, result.detach_called)
     end)
 
+    it('should detach buffer on bufwipe 2', function()
```

Comment:
any chance you could move this test to `test/functional/api/buffer_updates_spec.lua` and make it more targeted? it isn't related to LSP, and having a more targeted test should result in less code.